LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Park GiSung(minmul117)
```

Comment:
There are two problems:

- In English, there should be a space between a preceding word and parenthesis.
- `Name (username)` is not common in the license holder format. Use `Name <email>` instead.

Reply:
Okay, I changed which you suggested.